docs/documentation.md

```diff
@@ -1,371 +1,202 @@
 
-    ```shell
-	git status
-	```
+In addition, the coremltools repository includes the following:
 
-3. Switch to the `docs` folder and run `make clean` to delete any old HTML files (no effect if there are no HTML files):
+* [Core ML Tools README](https://github.com/apple/coremltools/blob/main/README.md) file for this repository.
+* [Release Notes](https://github.com/apple/coremltools/releases/) for the current release and previous releases.
 
-    ```shell
-	cd docs
-	make clean
-	```
 
-4. Change to the `coremltools` root folder (parent of `docs`), and run the following script to generate the HTML:
+## Updating the Guide and Examples Pages
 
-	```shell
-	cd ..
-	zsh ./scripts/build_docs.sh
-	```
+The Markdown files for [Guide and Examples](https://apple.github.io/coremltools/docs-guides/index.html) pages are organized in the `docs-guides/source` folder. The `docs-guides/index.rst` file provides the table of contents and left-column navigation. To make editing changes, follow these general steps:
 
-	*Warning*: Include the `./` or the script may not function correctly.
+1. Fork and clone the repository and create a branch for your changes.
 
-	The HTML files appear in the `docs/_build/html` folder.
+2. To preview the HTML, in the `docs-guides` folder enter `make clean` followed by `make html`. The Sphinx-generated HTML files appear in the `docs-guides/_build/html` folder. Open the preview by double-clicking `docs-guides/_build/html/index.html`.
 
-5. Preview your changes by right-clicking the `index.html` file in the `html` folder and choosing **Safari** (or another browser).
+3. When satisfied with your preview, copy the `html` folder to a temporarily location (outside of the `coremltools` clone controlled by git).
 
-6. To make more changes repeat Steps 2-4.
+4. Commit and push your changes.
 
-7. If satisfied with the preview from Step 5, add your changes for the commit.
+5. Create and submit a pull request as described in [Contributions](https://apple.github.io/coremltools/docs-guides/source/how-to-contribute.html#contributions) and add the **docs** label to it (see [Labels](https://apple.github.io/coremltools/docs-guides/source/how-to-contribute.html#labels) for details).
 
+6. When the pull request is approved and merged, switch to the root folder (`coremltools`), check out the `main` branch, and then checkout the `gh-pages` branch from upstream:
+    
     ```shell
-    git add --all
+    git checkout main
+    git checkout -b gh-pages upstream/gh-pages
     ```
 
-8. Make the commit.
+7. Copy the entire contents of your new  `html` folder from Step 3 to the Guide and Examples directory (`docs-guides`), overwriting any existing files and folders with the same names.
 
+8. Add and commit your changes, and push your changes to *your* fork:
+    
     ```shell
-    git commit -m "Commit message"
+    git add --all
```

Comment:
`git add --all` will add all files, even new/untraced file. I don't think we should tell users to blindly add all file.

Reply:
Removing this section about `gh-pages`.